src/App.tsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
 import './App.css';
 
+import { sampleList } from './sampleList';
+
+function ListDisplay() {
+  let items: JSX.Element[] = [];
+  for (let i = 0; i < sampleList.length; i++) {
+    let displayName = sampleList[i].name;
+    let item =
+      <div key={i}>
+        <span>
+          {displayName}
+        </span>
+      </div>
```

Comment:
I believe you only need the inner `span`. So it would become something like
```suggestion
        <span key={i}>
          {displayName}
        </span>
```

Nice work on using the `key` prop for this!